src/Pages/EventMain/services/getFilteredEvents.js

```diff
@@ -1,8 +1,17 @@
 export function getFilteredEvents(
   eventArray,
```

Comment:
When you have a function with so many parameters it might be helpful to pass an object and then destructure it. That way you don't need to remember the order of the arguments.